internal/driver/webui_test.go

```diff
@@ -249,3 +249,49 @@ func TestIsLocalHost(t *testing.T) {
 		}
 	}
 }
+
+func TestFlameGraph(t *testing.T) {
+	prof := makeFakeProfile()
+	ui := &webInterface{
+		prof:    prof,
+		options: &plugin.Options{Obj: fakeObjTool{}},
+		help:    make(map[string]string),
+	}
+
+	// Start test server.
+	server := httptest.NewServer(http.HandlerFunc(
+		func(w http.ResponseWriter, r *http.Request) {
+			ui.flamegraph(w, r)
+		}))
+	defer server.Close()
+
+	type testCase struct {
+		path string
+		want []string
+	}
+	testcases := []testCase{
```

Comment:
Page content is different and the tests are looking for different strings. Most of the strings are not present in TestWebInterface and vice-versa.